Gemfile_1.9

```diff
@@ -0,0 +1,11 @@
+source 'https://rubygems.org'
+
+gemspec
+
+group :test do
+  gem 'rubocop', "~> 0.41.0"
+  gem 'rake', '>= 2.4.2'
+  gem 'rspec'
+  gem 'simplecov', "~> 0.11.2"
```

Comment:
So this version is the last one supporting `1.9`?

Reply:
Yupp, couldn't find this in the RubyGems page, but as discussed offline, (and seeable through my commits haha) this is the most up to date version that works with 1.9.3